external-crates/move/crates/move-analyzer/tests/completion/sources/uid.move

```diff
@@ -1,5 +0,0 @@
-module Completion::uid {
```

Comment:
This file is seemingly not used anymore in tests. I think it was used to test object snippet insertion which is now done using `object.move` that additionally checks that resilient parsing works correctly for the snippet to be inserted